.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Is this `_` wanted ?

Reply:
I don't know why Husky always create this file on precommit :(

Reply:
OK, just wanted to make sure it wasn't a typo!